### ISSUE TITLE
Unable to manage file: SaltReqTimeoutError: after 180 seconds, ran 3 tries

### ISSUE THREAD

**Issue by abh23** [saltstack/salt#24962]
I am getting this error in one of minion and unable to run any command from master on the same. 

I have tried test.ping and state.apply for example. 

Salt-Master version : 2015.5.0 (Lithium)
Salt-Minion version :  2015.5.0 (Lithium)

**Comment by jtand** [comment]
@abh23 I'm not quite sure what you're asking for here. Could you provide the full debug log and the full command(s) you are running to get this error? Thanks!

**Comment by cachedout** [comment]
It sounds like the minion has lost connection to the master and is unable to re-establish it. Does restarting the minion process resolve the issue?

**Comment by abh23** [comment]
Ok. I got the fix. I had this network.managed for interfaces and were been called automatically from salt-master. They kind of changed the interfaces settings on minion. Due to this, minion wasn't able to reach master and hence the error. 

To debug, I removed minion's key from master and waited for it to re-accept which never happened. Going on level down to networking, I tried pinging salt master from minion and then I found the root cause. 

Thanks for replying though  !!
